test/src/module/storage/remoteTest.c

```diff
@@ -264,6 +264,24 @@ testRun(void)
             strZ(strNewBuf(storageGetP(storageNewReadP(storagePgWrite, STRDEF("test.txt"))))), FileMissingError,
             "raised from remote-0 shim protocol: " STORAGE_ERROR_READ_MISSING, TEST_PATH "/pg256/test.txt");
 
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("raw read file without compression");
+
+        HRN_STORAGE_PUT(storageTest, TEST_PATH "/repo128/test.txt", contentBuf);
+
+        // Disable protocol compression in the storage object
+        ((StorageRemote *)storageDriver(storageRepo))->compressLevel = 0;
+
+        StorageRead *fileReadRaw = NULL;
+        Buffer *buffer = bufNew(bufSize(contentBuf));
```

Comment:
Lets use bufUsed(contentBuf) here, because the test.txt file size can be accurately get with bufUsed. Now bufUsed == bufSize, but someday it may change